proof/crefine/ARM_HYP/AutoCorresTest.thy

```diff
@@ -1,636 +0,0 @@
-(*
- * Copyright 2014, General Dynamics C4 Systems
- *
- * SPDX-License-Identifier: GPL-2.0-only
- *)
-
-(* Experimental AutoCorres proofs over CRefine: work in progress *)
-
-theory AutoCorresTest
-imports Refine_C
-begin
-
```

Comment:
I should have a closer look at that.

Reply:
Ok, this was just overlooked. I think this happened because RISC-V got merged into the update somewhere in the middle after that theory had already been moved. Will be removed after the rebase.